state/machine.go

```diff
@@ -2030,7 +2030,7 @@ func (m *Machine) UpdateMachineSeries(series string, force bool) error {
 	return errors.Annotatef(err, "cannot update series for %q to %s", m, series)
 }
 
-func (m *Machine) verifyUnitsSeries(unitNames []string, series string, force bool) ([]*Unit, error) {
+func (m *Machine) VerifyUnitsSeries(unitNames []string, series string, force bool) ([]*Unit, error) {
```

Comment:
Does this need to be exported? If it is only exported for test, then perhaps you can move the tests into the internal test package instead.

Reply:
it's exported for use by the machinemanager in the apiserver